identity-account/src/storage/stronghold.rs

```diff
@@ -98,6 +99,19 @@ impl Storage for Stronghold {
     Ok(public)
   }
 
+  async fn key_insert(&self, id: IdentityId, location: &KeyLocation, secret_key: SecretKey) -> Result<PublicKey> {
+    let vault = self.vault(id);
+
+    vault
+      .insert(location_skey(location), secret_key.as_ref(), default_hint(), &[])
```

Comment:
Do we always want a blank `RecordHint`? Would setting it to the key type , e.g. `"Ed25519"`, be useful at all or does it introduce a security risk, or is that information essentially in the location anyway? 

Just asking out of interest, not suggesting a change.